2 - ListsLogicAndIterations/Code/LogicAndInterations.cs

```diff
@@ -1,36 +1,91 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ListsLogicAndIterations.Code
 {
 	public static class LogicAndInterations
 	{
 		public static int For_SumTwoListsAndAllValueThatAreNotFour(int[] numberArray, List<int> numberList)
 		{
-			throw new NotImplementedException();
-		}
+            int result = 0;
+			foreach(int num in numberArray)
+            {
+                if (num != 4)
+                {
+                    result += num;
+                }
+            }
+            result += ForEach_SumTwoListsAndAllValueThatAreNotFour(numberList);
+            return result;
```

Comment:
Though I understand why you did this, the goal was to implement the `For` statement, not a for-each statement. They are two very different beasts and can accomplish different tasks.